sqldelight-compiler/integration-tests/src/test/kotlin/com/example/testmodule/TestDatabaseImpl.kt

```diff
@@ -0,0 +1,308 @@
+package com.example.testmodule
+
+import com.example.GroupQueries
+import com.example.Player
+import com.example.PlayerQueries
+import com.example.SelectNull
+import com.example.Team
+import com.example.TeamQueries
+import com.example.TestDatabase
+import com.squareup.sqldelight.Query
+import com.squareup.sqldelight.TransacterImpl
+import com.squareup.sqldelight.core.integration.Shoots
+import com.squareup.sqldelight.db.SqlCursor
+import com.squareup.sqldelight.db.SqlDriver
+import java.lang.Void
+import kotlin.Any
+import kotlin.Int
+import kotlin.Long
+import kotlin.String
+import kotlin.collections.Collection
+import kotlin.collections.MutableList
+
+internal object TestDatabaseImplExposer {
```

Comment:
Can we nuke this object and use extension properties?

```kotlin
internal val TestDatabase.schema: SqlDriver.Schema get() = TestDatabaseImpl.Schema

internal operator fun TestDatabase.invoke(..): TestDatabase = TestDatabaseImpl(..)
```

Reply:
im going to merge this first, then remove in a separate PR to keep things simple

Reply:
ehhh its not that easy, they need to be companion functions and not on an instance

Reply:
`internal val KClass<TestDatabase>.schema` and `TestDatabase::class.schema`?

You also could just make it a top-level property like `internal val testDatabaseSchema`

Or put an empty companion in the interface and hang the extension off of it: `internal val TestDatabase.Companion.schema`

Reply:
i like the kclass idea.